doc/tutorials/pre-launched-rt.rst

```diff
@@ -106,3 +105,14 @@ Reboot the system, and it will boot into Pre-Launched RT Mode
    26c5e0d88f8a47d88109f201ebd61a5e   0   ACRN PRE-LAUNCHED VM0            Running
    dbbbd4347a574216a12c2201f1ab0240   1   ACRN SOS VM                      Running
    ACRN:\>
+
+Connect console of VM0, via 'vm_console' ACRN shell command (Use 'Ctrl + Alt + Space' key
```

Comment:
can you verify if this should be CTRL + SPACE  (no ALT needed)?

Reply:
(CTRL + SPACE) is right

Reply:
Fixed